app/dts/layouts/common/40percent/40percent.dtsi

```diff
@@ -0,0 +1,53 @@
+#include <layouts/common/40percent/1x2u.dtsi>
```

Comment:
I don't really like the naming for this nor the 50percent folder... Although 60%, 65%, etc. have a fairly standard small set of standard layouts, the world of 50 and 40% keyboards has way too much variation, and trying to make catch all folders for these seems fraught.

I'd rather name thes something like `ortho_5x12` and `ortho_4x12`. Thoughts?

Reply:
Two ways we could do this:

1. Still under the `40percent` folder, but have sub-aggregates like `40percent/ortho_4x12.dtsi` that pulls in `40percent/ortho_4x12_ortho.dtsi`, `40percent/ortho_4x12_mit.dtsi`, etc.
1. Have separate common folders, e.g. `common/ortho_4x12/` and `common/ortho_5x12/`.

Reply:
Maybe separate common folders is a better approach otherwise the 40 and 50% folders may get very cluttered with all sorts of layouts?

Reply:
> Maybe separate common folders is a better approach otherwise the 40 and 50% folders may get very cluttered with all sorts of layouts?

WFM.

Reply:
What's your opinion on the individual layout node names, is `&layout_40_xxxx` still good, or should I change it to `&layout_ortho_4x12_xxxx`. I can envision a scenario where a 40 board has both ortholinear and staggered layouts so there may be a naming conflict. Perhaps it's better to be fully specific about the name?

Reply:
I like the more specific, definitely.